docker/calamari.Dockerfile

```diff
@@ -32,8 +32,8 @@ RUN $PARA_BINARY_PATH --version
 ADD $PARA_GENESIS_URL $PARA_GENESIS_PATH
 ADD $RELAY_GENESIS_URL $RELAY_GENESIS_PATH
 
-# Expose three ports by default
-EXPOSE 30333 9933 9944
+# Expose 5 ports by default
+EXPOSE 30333 30334 9933 9944 9615
```

Comment:
Also, I'm not sure hardcoding the expose in the dockerfile is the way to go, this way it won't be possible to run two containers on the same machine in parallel because the first one will claim the host ports.

I would recommend not having an expose section in the Dockerfile and instead ask the user to invoke 
`docker run -p 30333:30333 -p 30333:30334 <+other needed ports> imagename`

That way the second container can be invoked with 
`docker run -p 30333:30343 -p 30333:30344 <+other needed ports> imagename`
and peering will still work.

Reply:
> Exposing ports is a way of documenting which ports are used, but does not actually map or open any ports. Exposing ports is optional.
](https://docs.docker.com/engine/reference/builder/#expose)

resolving this